app/controllers/mixins/ems_common/angular.rb

```diff
@@ -357,6 +366,10 @@ def ems_form_fields
           host_default_vnc_port_end = @ems.host_default_vnc_port_end.to_s
         end
 
+        if @ems.supports?(:assume_role)
+          service_account = @ems.authentication_service_account
```

Comment:
this will break if `ManageIQ::Providers::Google::CloudManager` ever decides to `supports :assume_role`

Because GCE is already using `service_account` to mean something else.

If these 2 are the same concept, shouldn't they be treated the same? (and use `layouts/angular/auth_service_account_angular`)

But it seems more likely that those 2 are not related at all, so I think you shouldn't be using the `service_account` variable to mean 2 different things.

Reply:
Those are different things, we discussed with @agrare and agreed to avoid new db field. GCE is not broken, because I didn't touch anything related to it. You can see it's lines above mine I've added.

Reply:
Nothing to do with DB fields.

You have a local variable called `service_account` in this `ems_form_fields` function.
That gets sent as `data.service_account`, but depending on the provider, the value comes from `@ems.authentication_service_acount` **or** from `@ems.authentication_token`.

This local variable ends up as `data.service_account` in the angular JS controller, where it gets used 2 different ways, in 2 different unrelated fields.